app/src/main/res/layout/bottomsheet_attendee_check_in.xml

```diff
@@ -144,7 +144,7 @@
                         android:layout_height="wrap_content"
                         android:layout_gravity="center_vertical"
                         android:padding="@dimen/spacing_extra_small"
-                        android:text="@{ DateUtils.formatDateWithDefault(DateUtils.FORMAT_DAY_COMPLETE, checkinAttendee.order.completedAt) }"
+                        android:text="@{ checkinAttendee.order.completedAt != null ? DateUtils.formatDateWithDefault(DateUtils.FORMAT_DAY_COMPLETE, checkinAttendee.order.completedAt) : checkinAttendee.order.completedAt }"
```

Comment:
Progress Bar won't be even visible to the user. It'll take milliseconds to get bound, but in case it s actually null, we should use a message for user.